PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1

```diff
@@ -0,0 +1,39 @@
+# Initialize the context
+.\Init.ps1
+
+# Here are 3 common ways to retrieve entities
+# 1. Retrieve all the customers in a workgroup and process them one by one
+# Use -RetrieveAll and piping with ForEach 
+Get-BT_Customer -Ticket $mspc.WorkgroupTicket -RetrieveAll | ForEach {
+    # Process each customer retrieved
+    Write-Host $_.CompanyName
+}
+
+# 2. Retrieve and process customers from a workgroup in batches
+# Use paging
+$pageSize = 100
+$count = 0
+While( $true ) {   
+    # Retrieve customers in batch
+    [array]$batch = Get-BT_Customer -Ticket $mspc.WorkgroupTicket -PageOffset $($count * $pageSize)
```

Comment:
Same here and below, let's use Get-BT_Endpoint + $mspc.CustomerTicket for this sample script.

Reply:
OK